frontend/packages/kubevirt-plugin/src/components/vms/vm-users.tsx

```diff
@@ -0,0 +1,152 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { watchURL, stopWatchURL } from '@console/internal/actions/dashboards';
```

Comment:
dont use dashboards stuff outside of dashboards. It would be good to create a new hook for polling some url. You can take a look at https://github.com/openshift/console/blob/master/frontend/public/components/graphs/prometheus-poll-hook.ts - create something generic and reuse it also in prometheus-poll-hook

Reply:
Thanks ! using pool hook now